src/featurecounts/script.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+## VIASH START
+## VIASH END
+
+if [[ $par_tmpdir ]] && [[ ! -d "$par_tmpdir" ]]; then
+    mkdir -p $tmpDir
+fi
```

Comment:
Use `$meta_temp_dir` to create a temporary directory

```suggestion
tmp_dir=$(mktemp -d -p "$meta_temp_dir" "${meta_functionality_name}_XXXXXXXXX")
```